### PR TITLE
Add a new category: A Little Bit of Everything! 😃

### PR DESCRIPTION
**Pull request by Raymo111**
## What type of PR is this? (check all applicable)

- [x] 🚀 Added Name
- [x] ✨ Feature
- [ ] ✅ Joined Community
- [ ] 🌟 ed the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

### CODE REVIEW

**Comment by Raymo111** [comment]
@abhisheknaiidu I've moved the section lower down!

File: README.md [hunk 1]
```diff
@@ -20,10 +20,12 @@
 
 ### Contents:
   - [Categories](#categories)
+      - [A little bit of everything 😃](#a-little-bit-of-everything-)
```
Comment (abhisheknaiidu) [inline README.md:23]: remove this

File: README.md [hunk 2]
```diff
@@ -42,6 +44,9 @@
 
 ## Categories
 
+#### A Little Bit of Everything 😃
+- [Raymond Li](https://github.com/Raymo111/Raymo111)
+
```
Comment (abhisheknaiidu) [inline README.md:49]: Remove these

**Comment by Raymo111** [comment]
@abhisheknaiidu I have no idea how those got there! I must have merged instead of replaced (the patch-1 thing is kinda confusing), but it should be good now. Sorry for the inconvenience and thanks!

File: README.md [hunk 3]
```diff
@@ -24,6 +24,7 @@
       - [Game Mode 🚀](#game-mode-)
       - [Code Mode 👨🏽‍💻](#code-mode-)
       - [Dynamic Realtime 💫](#dynamic-realtime-)
+      - [A little bit of everything 😃](#a-little-bit-of-everything-)
```
Comment (abhisheknaiidu) [inline README.md:27]: Change `A little bit of everything 😃` to `A Little Bit of Everything 😃`

**Comment by Raymo111** [comment]
@abhisheknaiidu done!

**Comment by Raymo111** [comment]
@abhisheknaiidu Thanks!

**Pull request merged**